Dockerfile.base

```diff
@@ -0,0 +1 @@
+FROM microsoft/aspnetcore-build:2.0
```

Comment:
It's kind of weird to use an ASP.NET docker image to build something that's not a webserver. Could we use `microsoft/dotnet` instead?

Reply:
Yes! It makes more sense to use dotnet, that one was the one that I found 1st and as a not very experienced docker user, tried that one, microsoft/dotnet seems to work fine as well just tested it!